core/templates/update.use-cli.php

```diff
@@ -1,11 +1,11 @@
 <div class="update" data-productname="<?php p($_['productName']) ?>" data-version="<?php p($_['version']) ?>">
 	<div class="updateOverview">
-		<h2 class="title"><?php p($l->t('Update needed')) ?></h2>
+		<h2 class="title"><?php p($l->t('Upgrade needed')) ?></h2>
 		<div class="infogroup">
 			<?php if ($_['tooBig']) {
 	p($l->t('It looks like your instance may host many files and/or users. To ensure a smooth upgrade process, please use the command line updater (occ upgrade).'));
```

Comment:
I guess this line is OK - "please use the command line updater (occ upgrade)" 
"command line upgrader" sounds strange to me in English.

Reply:
please use the command line upgrade command instead?

Reply:
```suggestion
	p($l->t('It looks like your instance may host many files and/or users. To ensure a smooth upgrade process, please use the command line upgrade command (occ upgrade).'));
```

@jnweiger what do you think should be written here?